PWGHF/HFC/Tasks/taskCorrelationD0Hadrons.cxx

```diff
@@ -215,41 +262,77 @@ struct HfTaskCorrelationD0Hadrons {
         continue;
       }
       // check if correlation entry belongs to signal region, sidebands or is outside both, and fill correlation plots
-      if ((massD > signalRegionInner->at(ptBinD) && massD < signalRegionOuter->at(ptBinD)) && ((signalStatus == 1) || (signalStatus == 3))) {
+      if ((massD > signalRegionInner->at(ptBinD) && massD < signalRegionOuter->at(ptBinD)) && ((signalStatus == ParticleTypeData::isOnlyD0) || (signalStatus == ParticleTypeData::isBothD0D0bar))) {
```

Comment:
What do `signalRegionInner` and `signalRegionOuter` mean?

Reply:
These two are the lower and upper range for D0 mass peak. Same naming convention is followed by other committed tasks in HFC.

Reply:
I find it very confusing to use `Inner` and `Outer` instead of `Lower` and `Upper` for the signal range as the edges are not inside or outside w.r.t. the peak mean.
@fcolamar Is this the standard terminology?

Reply:
Indeed the other tasks also used inner and outer (though lower and higher probably sounds better), anyway @samrangy consider that you should swap your sidebandLeftInnerDefault and sidebandLeftOuterDefault array names (you now have as 'outer' the values closer to the peak, and viceversa)

Reply:
Thanks @fcolamar for the feedback and for spotting the wrong default values. Since the terminology seems to be based on the position w.r.t. the peak mean (you have left and right sidebands, each with the outer and inner edges), I think it might fit in this scheme best, if you call the signal region edges `signalRegionLeft` and `signalRegionRight`. Would that sound more natural?

Reply:
@vkucera I agree with your suggestion and modified accordingly.
@fcolamar Thanks for pointing out, I have swapped the arrays.